.changeset/unlucky-deers-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sei-js/react': minor
```

Comment:
actually can u update this to be a patch change? think it makes more sense than minor